src/views/admin/PersonDetailsView.vue

```diff
@@ -1683,6 +1695,7 @@
                   :errorCode="personStore.errorCode"
                   :person="personStore.currentPerson"
                   :adminId="authStore.currentUser?.personId!"
+                  :intersecting-organisations="intersectingOrganisations"
```

Comment:
```suggestion
                  :intersectingOrganisations="intersectingOrganisations"
```
Sollte auch gehen und konform sein mit was bisher gemacht wurde.